### DISCUSSION TITLE
Misspelling in Github Copilot settings

### DISCUSSION THREAD

**Discussion by Cyborus04** [community/community#21404] [Other Feature Feedback, Questions, & Ideas]
<p>There’s a minor typo in the Github Copilot settings</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/0/208fac48af0e2b29653c040a586c0ab419c21b33.png" data-download-href="/uploads/short-url/4E360Syhk8zdJLsA7FeHJB515vR.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/87248184/181064428-4bcc812d-1d27-4d9f-985d-da62bf7d8933.png" alt="image" data-base62-sha1="4E360Syhk8zdJLsA7FeHJB515vR" width="690" height="73" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/0/208fac48af0e2b29653c040a586c0ab419c21b33_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">887×95 2.61 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>It says “blow” where it likely should say “below”</p>
<p><small>not sure if this is the right place to report this, I’m sorry if it isn’t</small></p>

**Comment by Akash1134** [comment, marked as answer]
<p>Hi there! Visit the <a href="https://github.com/github-community/community/discussions/categories/copilot">Feedback forum</a> to ask questions, share bugs or feedback, or chat with other users in the Preview. The GitHub Copilot team will respond as often as possible, but we also welcome you to share your experiences and help others in the community.</p>
